tests/periph_rtt/Makefile

```diff
@@ -2,6 +2,7 @@ BOARD ?= samr21-xpro
 include ../Makefile.tests_common
 
 FEATURES_REQUIRED = periph_rtt
+FEATURES_OPTIONAL += periph_rtt_set_counter
```

Comment:
I think here we need a Kconfig to `imply` this module when it's available. Something like

https://github.com/RIOT-OS/RIOT/blob/dfb2c21570eb31bd9003eff20e1a7d2d1365c7d4/tests/periph_flashpage/Kconfig#L8-L13

Reply:
Thanks. This did the trick (at least locally it seems to work now). I directly amended the change into the last commit.